Library/Formula/truncate.rb

```diff
@@ -0,0 +1,34 @@
+class Truncate < Formula
+  desc "truncates a file to a given size"
+  homepage "http://www.vanheusden.com/truncate"
+  url "http://www.vanheusden.com/truncate/truncate-0.8.tgz"
+  sha256 "165914d643351324d69096427d3ce5d3ab13aed3f7c990a71ba3bd0c6f278f00"
+  patch :DATA # remove missing file; rename syscalls for OSX which are natively 64bit
```

Comment:
Please leave an empty line between the initial block and any patching. The need to patch for build success also needs to be reported upstream.